src/OrleansRuntime/Silo/Silo.cs

```diff
@@ -244,6 +244,7 @@ internal Silo(SiloInitializationParameters initializationParams, IServiceProvide
 
             // GrainRuntime can be created only here, after messageCenter was created.
             grainRuntime = Services.GetRequiredService<IGrainRuntime>();
+            StreamProviderManager = Services.GetRequiredService<IStreamProviderManager>();
```

Comment:
I'm not convinced we need StreamProviderManager as member in silo any more, but removing it can easily be considered out of scope for this PR.  Up 2 u. :)

Reply:
Yes it is something I was considering... But I prefer keeping changes minimal for now, I will remove that after beta if it is ok for you